### ISSUE TITLE
Ability to exclude (hide) specific files/directories from displaying in FSBrowser

### ISSUE THREAD

**Issue by NShaforostov** [epam/cloud-pipeline#854]
**Background**
At the moment, Cloud Pipeline displays all node files/directories (according to the specified work directory). But that "full" user access to all node content could be unsafety. In certain cases, it would be convenient for admins to have the ability to hide specific files or directories.

**Approach**
Admins should have the ability to specify which node files/directories shall be hidden for displaying via FSBrowser.
A new system preference should be added into the "Data storage" section to expand the FSBrowser behavior, e.g.: `storage.fsbrowser.black.list`. Which shall behave in the same manner as `storage.mount.black.list`:
- Type: string
- Default value: `/bin,/var,/home,/root,/sbin,/sys,/usr,/boot,/dev,/lib,/proc`
- Validation: can be empty
- Visibility: false

This preference shall control, which paths are "hidden" from the output to the client. Any request to the "hidden" path - shall be restricted.
File(directory) names shall be specified using a comma.

**Comment by NShaforostov** [comment]
@ekazachkova, I tried to verify this, the following issues were found:
1. The feature doesn't work if the specified path is started from spaces. E.g. if the following: `/bin, /var` specify into `storage.fsbrowser.black.list` then `/bin` directory will be excluded but `/var` will be available
2. The feature doesn't work if the specified path is a relative path. E.g. if the following: `/bin,var` specify into `storage.fsbrowser.black.list` then `/bin` directory will be excluded but `/var` will be available. I suggest if the specified path is relative then it should be excluded relatively to the `storage.fsbrowser.wd` value
3. The feature didn't work when I specified the path like `/cloud-data/<storage_name>` (where storage_name is the existing storage name) into `storage.fsbrowser.black.list` (see more details in #882)
